projects/manage-supervision-and-delius/src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/SentenceIntegrationTest.kt

```diff
@@ -67,8 +67,8 @@ class SentenceIntegrationTest {
                     Order("Default Sentence Type", 12, null, LocalDate.now().minusDays(14)),
                     listOf(Requirement("Main", "High Intensity", 12, "my notes", Rar(1, 0, 1))),
                     listOf(
-                        CourtDocument(115, LocalDate.now().minusDays(1), "court report"),
-                        CourtDocument(116, LocalDate.now().minusDays(3), "event report")
+                        CourtDocument(118, LocalDate.now().minusDays(1), "court report"),
```

Comment:
It looks like these CourtDocument entities are generated here: https://github.com/ministryofjustice/hmpps-probation-integration-services/blob/main/projects/manage-supervision-and-delius/src/dev/kotlin/uk/gov/justice/digital/hmpps/data/DataLoader.kt#L75

It might be cleaner if we moved those to some static vals (e.g. in a CourtDocumentGenerator), so we could reference them here, rather than updating these ids each time?